fabfile.py

```diff
@@ -208,22 +208,37 @@ def docker_compose_run(command_name, service="builder", user="app", no_deps=Fals
 
 def set_local_configuration():
     env.compose_files = ['docker-compose.yml']
-    env.user_id = int(local('id -u', capture=True))
-    env.root_dir = os.path.dirname(os.path.abspath(__file__))
+    env.dinghy = False
+    env.powerShell = False
```

Comment:
We are using snake_case for var name. Could you update `powerShell` to `power_shell`? Thanks